service/component-samples/reseller/src/main/java/org/fido/iot/sample/ResellerVoucherServlet.java

```diff
@@ -77,8 +77,6 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp)
     }
 
     Composite voucher = result.getAsComposite(Const.FIRST_KEY);
-    List<PublicKey> certs =
```

Comment:
Please keep this as this will be needed for fixing an issue.

Reply:
Okay